includes/gateway/class-omise-payment.php

```diff
@@ -226,6 +226,62 @@ public function process_payment( $order_id ) {
 	abstract public function result( $order_id, $order, $charge );
 
 	/**
+	 * Process refund.
+	 *
+	 * @param  int    $order_id
+	 * @param  float  $amount
+	 * @param  string $reason
+	 *
+	 * @return boolean True|False based on success, or a WP_Error object.
+	 *
+	 * @see    WC_Payment_Gateway::process_refund( $order_id, $amount = null, $reason = '' )
+	 */
+	public function process_refund( $order_id, $amount = null, $reason = '' ) {
+		if ( ! $order = wc_get_order( $order_id ) ) {
+			$message = __(
+				'Refund failed. Cannot retrieve an order from the given ID: %s. Please try again or you may do a manual refund.', 
```

Comment:
"... with the given ID: ..."

Reply:
"...or do a manual refund."

Reply:
Thanks 👍 
Wonder what's the difference between `or you may do a manual refund` and `or do a manual refund`. Just a nuanced or it gives a totally different impression?

Reply:
"Please try again or you may do a manual refund" is kinda weird sounding. It also has a subtly different meaning overall, which I'm struggling to articulate - something like "if you don't try again, you may inadvertently do a manual refund" - which isn't the intended message